### PR TITLE
CA-169167: Screen resolution is less than the default value after switch to remote desktop

### PR DESCRIPTION
**Pull request by cheng--zhang**
Auto switch to RDP should be taken after VNC connection got the correct screen resolution.
Also not enable “enable RDP button” before VNC connection get the correct screen resolution.

Signed-off-by: Cheng Zhang <cheng.zhang@citrix.com>

### CODE REVIEW

**Comment by Zhengchai** [comment]
Hi There - this one has been a while for review, could you help have a look please?

File: XenAdmin/ConsoleView/XSVNCScreen.cs [hunk 1]
```diff
@@ -588,6 +589,29 @@ private void ConnectToRemoteConsole()
...
+            {
+                if (OnDetectRDP != null)
+                    Program.Invoke(this, OnDetectRDP);
+                AutoSwitchRDPLater = false;
+            }
+            if (Helpers.CreamOrGreater(Source.Connection) && parentVNCTabView.IsRDPControlEnabled())
+                parentVNCTabView.EnableToggleVNCButton();
+        }
+
+        internal bool AutoSwitchRDPLater
```
Comment (MihaelaStoica) [inline XenAdmin/ConsoleView/XSVNCScreen.cs]: As no additional logic is not required here, this property can be auto-implemented, to make the declaration more concise, like:
internal bool AutoSwitchRDPLater { get; set; }

**Comment by MihaelaStoica** [comment]
Apart from the note above, everything looks good.

**Comment by cheng--zhang** [comment]
Hi @MihaelaStoica 
Thanks a lot for your comments. I already change the code. Please help to review the code again.

**Comment by MihaelaStoica** [comment]
Thanks @cheng--zhang . Merging.

**Pull request merged**